web/src/buddy_data.ts

```diff
@@ -464,6 +466,10 @@ export function get_conversation_participants(): Set<number> {
             participant_ids_set.add(message.sender_id);
         }
     }
+    const conversation_participants = new ConversationParticipants(
+        message_lists.current.all_messages(),
+    ).visible();
+    assert(_.isEqual(participant_ids_set, conversation_participants));
```

Comment:
This needs to be a blueslip error rather than an assert, or it'll break the app when it fails.